src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
The stylesheet link references an `.scss` file. Browsers cannot directly process SCSS files; they need to be compiled into CSS first. Ensure that the SCSS file is compiled into a CSS file and update the `href` attribute to point to the resulting CSS file.